sentinel/package.json

```diff
@@ -11,6 +11,7 @@
     "async": "^3.2.4",
     "bikram-sambat": "^1.7.0",
     "bikram-sambat-bootstrap": "^1.5.0",
+    "cron-validator": "^1.3.1",
```

Comment:
As above, I _think_ it's sufficient to add this as a dependency in `shared-libs/transitions`, where it will end up being used.

Reply:
Same case as [this](https://github.com/medic/cht-core/pull/8387#discussion_r1340196350) comment

Reply:
You're right! Apparently, with workspaces, you no longer store dependencies in workspaces :(